storage/mysql/storage.sql

```diff
@@ -83,6 +83,8 @@ CREATE TABLE IF NOT EXISTS LeafData(
   -- This is extra data that the application can associate with the leaf should it wish to.
   -- This data is not included in signing and hashing.
   ExtraData            LONGBLOB,
+  -- The timestamp from when this leaf data was first queued for inclusion.
+  QueueTimestampNanos  BIGINT NOT NULL,
```

Comment:
The NOT NULL would be ideal, but given the problems it might cause with existing installations I think we should drop it.

Reply:
I don't think it causes problems because of the implicit default value...